ImportError with version 0.36.0

**Describe the bug**

Error trying to use the 0.36.0 release:

```
ImportError while loading conftest '/home/travis/build/IBM-Q-Software/multi-channel-scheduler-integration-tests/tests/conftest.py'.
tests/conftest.py:5: in <module>
    from qiskit_ibm_runtime import QiskitRuntimeService, Session, Batch
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/__init__.py:205: in <module>
    from .qiskit_runtime_service import QiskitRuntimeService
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/qiskit_runtime_service.py:26: in <module>
    from qiskit_ibm_runtime import ibm_backend
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/ibm_backend.py:44: in <module>
    from .api.clients import RuntimeClient
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/api/clients/__init__.py:15: in <module>
    from .base_websocket_client import WebsocketClientCloseCode
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/api/clients/base_websocket_client.py:27: in <module>
    from ..client_parameters import ClientParameters
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/api/client_parameters.py:18: in <module>
    from ..utils import default_runtime_url_resolver
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/utils/__init__.py:38: in <module>
    from .json import RuntimeEncoder, RuntimeDecoder, to_base64_string
.tox/staging/lib/python3.9/site-packages/qiskit_ibm_runtime/utils/json.py:91: in <module>
    int(x) for x in re.match(r"\d+\.\d+\.\d", _aer_version_string).group(0).split(".")[:3]
E   NameError: name '_aer_version_string' is not defined
```

Looks like that's due to PR #2124.

**Steps to reproduce**

Install and import from 0.36.0.

**Expected behavior**

There shouldn't be an ImportError.

**Suggested solutions**

Looks like aer needs to be optional if it's not installed with qiskit-ibm-runtime, this is what I have installed:

>annotated-types==0.7.0,certifi==2025.1.31,cffi==1.17.1,charset-normalizer==3.4.1,cryptography==44.0.1,dill==0.3.9,exceptiongroup==1.2.2,ibm-cloud-sdk-core==3.22.1,ibm-platform-services==0.59.1,idna==3.10,iniconfig==2.0.0,mpmath==1.3.0,multi-channel-scheduler-integration-tests @ file:///home/travis/build/IBM-Q-Software/multi-channel-scheduler-integration-tests/.tox/.tmp/package/1/multi_channel_scheduler_integration_tests-0.0.1.tar.gz#sha256=04d3d716d18ba6e902983f5b1e366abbf6df5fc07fc4dde5487fec41baaac3f0,numpy==2.0.2,packaging==24.2,pbr==6.1.1,pip==25.0.1,pluggy==1.5.0,pycparser==2.22,pydantic==2.9.2,pydantic_core==2.23.4,PyJWT==2.10.1,pyspnego==0.11.2,pytest==8.3.4,python-dateutil==2.9.0.post0,qiskit==1.3.2,qiskit-ibm-runtime==0.36.0,requests==2.32.3,requests_ntlm==1.3.0,rustworkx==0.16.0,scipy==1.13.1,setuptools==75.8.0,six==1.17.0,stevedore==5.4.0,symengine==0.13.0,sympy==1.13.3,tomli==2.2.1,typing_extensions==4.12.2,urllib3==2.3.0,websocket-client==1.8.0,wheel==0.45.1

**Additional Information**

- **qiskit-ibm-runtime version**: 0.36.0
- **Python version**: 3.9.18
- **Operating system**: Ubuntu Focal

--- a/qiskit_ibm_runtime/utils/json.py
+++ b/qiskit_ibm_runtime/utils/json.py
@@ -14,7 +14,6 @@
 
 """Utility functions for the runtime service."""
 
-import re
 import base64
 import copy
 import importlib
@@ -39,7 +38,6 @@ except ImportError:
 
 try:
     import qiskit_aer
-    from qiskit_aer.version import __version__ as _aer_version_string
 
     HAS_AER = True
 except ImportError:
@@ -86,10 +84,6 @@ from qiskit_ibm_runtime.utils.estimator_pub_result import EstimatorPubResult
 from .noise_learner_result import NoiseLearnerResult
 
 SERVICE_MAX_SUPPORTED_QPY_VERSION = 13
-
-_AER_VERSION = tuple(
-    int(x) for x in re.match(r"\d+\.\d+\.\d", _aer_version_string).group(0).split(".")[:3]
-)
 
 
 def to_base64_string(data: str) -> str:
@@ -508,7 +502,7 @@ class RuntimeDecoder(json.JSONDecoder):
             if obj_type == "to_json":
                 return obj_val
             if obj_type == "NoiseModel":
-                if HAS_AER and _AER_VERSION[1] >= 16:
+                if HAS_AER:
                     return qiskit_aer.noise.NoiseModel.from_dict(obj_val)
                 warnings.warn("Qiskit Aer is needed to restore noise model.")
                 return obj_val

--- a/release-notes/unreleased/2144.bug.rst
+++ b/release-notes/unreleased/2144.bug.rst
@@ -1,0 +1,2 @@
+Fixed an issue where having ``qiskit-aer`` not installed would 
+cause an import error.

